boskos/janitor/janitor.go

```diff
@@ -44,8 +71,12 @@ func main() {
 
 // Clean by janitor script
 func janitorClean(proj string) error {
-	script := "../../jenkins/janitor.py"
-	return exec.Command(script, fmt.Sprintf("--project=%s", proj), "--hour=0").Run()
+	cmd := exec.Command("/janitor.py", fmt.Sprintf("--project=%s", proj), "--hour=0")
+	b, err := cmd.CombinedOutput()
+	if err != nil {
+		logrus.Infof("janitor.py has some issue: %s", string(b))
```

Comment:
This error is propagated up and logged there. I use `Infof` here try to help better log filtering, since this message can be very long from janitor.